src/edu/macalester/comp127/FarmArt/FarmArtGame.java

```diff
@@ -0,0 +1,150 @@
+package edu.macalester.comp127.FarmArt;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.awt.Color;
+import edu.macalester.graphics.CanvasWindow;
+import edu.macalester.graphics.GraphicsGroup;
+import edu.macalester.graphics.Image;
+import edu.macalester.graphics.Rectangle;
+
+/**
+ * A farm art game that allows user to change background to water, dirt or grass
+ * also allows user to add elements, including vegetables, aniamls and flowers on top of each background
+ * also allows user to zoom in/out or drag the farm to create as big/small as they wish
+ * the farm art game uses element images made by Ned Mayo, Stefan Gullickson, Jiaying Wu
+ */
+public class FarmArtGame {
+
+    private static CanvasWindow canvas;
+    private TileManager tileManager;
+    private GraphicsGroup backdrop;
+    private Rectangle rectangle;
+    private Rectangle selectedButtonIndicator;
```

Comment:
This class and its instance variable names do a great job of explaining its purpose and structure…except for `rectangle`. What is `rectangle`?? (Answer: it doesn’t even need to be an instance variable at all!)